Core/NoOpRunner.Core/GamePlatforms.cs

```diff
@@ -53,6 +53,7 @@ public void ClickShape(int x, int y)
                         pixel.X >= SizeX ||
                         pixel.Y >= SizeY)
                     {
+                        //so?? just skip
```

Comment:
This should currently never happen. Thus, the exception is useful, because if it is thrown, we fucked something up.

Reply:
Happens on X axis,  'cuz generation in segments from 2 to 4 and can get over max X axis, should check after generating a segment and trunc if over max X pixels count